ref/gl/gl_image.c

```diff
@@ -1586,11 +1587,11 @@ int GL_LoadTextureArray( const char **names, int flags )
 	for( i = 0; i < numLayers; i++ )
 	{
 		COM_FileBase( names[i], basename );
-		Q_strncat( name, basename, sizeof( name ) );
-		if( i != ( numLayers - 1 )) Q_strncat( name, "|", sizeof( name ));
+		len += Q_strncpy( &name[len], basename, sizeof( name ) - len );
```

Comment:
Why not just keep `Q_strncat` for a while? There is too critical to be part of this PR.